framework/Web/UI/WebControls/TPanelStyle.php

```diff
@@ -103,6 +110,11 @@ public function addAttributesToRender($writer)
 				$this->setStyleField('direction', 'rtl');
 			}
 		}
+		if ($boxShadow = $this->getBoxShadow()) {
+			$this->setStyleField('-moz-box-shadow', $boxShadow);
+			$this->setStyleField('-webkit-box-shadow', $boxShadow);
+			$this->setStyleField('box-shadow', $boxShadow);
```

Comment:
Please drop the vendor-specific -moz-box-shadow and -webkit-box-shadow, they don't really add support for any browser

Reply:
will do.

Reply:
changed